workers_controller.js

```diff
@@ -14,6 +14,10 @@
 
 'use strict';
 
+if (process.env.NEW_RELIC_LICENSE_KEY) {
```

Comment:
We can have this condition within the `./helpers/newrelic_list` file as well.

Reply:
Explained earlier as well..